$NetBSD: patch-aa,v 1.2 2009/07/07 20:46:53 sno Exp $

Do not error out if perl was built without PERL_MALLOC_WRAP

--- stolen_chunk_of_toke.c.orig	2009-06-03 11:18:50.000000000 +0000
+++ stolen_chunk_of_toke.c	2009-06-22 07:23:46.000000000 +0000
@@ -48,7 +48,7 @@
 #define PTR2nat(p)       (PTRV)(p)       /* pointer to integer of PTRSIZE */
 #endif
 
-#ifndef MEM_WRAP_CHECK_
+#if defined(PERL_MALLOC_WRAP) && !defined(MEM_WRAP_CHECK_)
 #define MEM_WRAP_CHECK_(n,t) MEM_WRAP_CHECK(n,t),
 #endif
 
